src/assets/components/ui/AvtorCard/AvtorCard.tsx

```diff
@@ -0,0 +1,39 @@
+import style from "./AvtorCard.module.scss"
+import cn from "classnames"
+import Image from "@/assets/components/ui/Image"
+
+interface Props {
+  className?: string
+  number: string
+  src: string
```

Comment:
it's better to define avtor interface in src/types/avtor.ts and use it here like `avtor: Avtor`